.github/workflows/test_build_wheels_linux_with_cuda.yml

```diff
@@ -35,11 +35,11 @@ jobs:
             post-script: packaging/post_build_script.sh
             smoke-test-script: test/smoke_test.py
             package-name: torchvision
-          - repository: pytorch/torchrec
-            pre-script: ""
-            post-script: ""
-            smoke-test-script: ""
-            package-name: torchrec
+          # - repository: pytorch/torchrec
+          #   pre-script: ""
+          #   post-script: ""
+          #   smoke-test-script: ""
```

Comment:
Is this intentional commenting out?

Reply:
Intentional - our torchrec builds need to be fixed in general so leaving them out for now (also not in scope for 3.11)

Reply:
Ah, I recall that. Makes sense.